RoddyCore/src/de/dkfz/roddy/execution/io/fs/FileSystemAccessProvider.groovy

```diff
@@ -68,6 +68,11 @@ public class FileSystemAccessProvider {
      */
     protected File _userHome;
 
+    /**
+     * The cached "remote" umask
+     */
+    protected Integer _umask
```

Comment:
Then you call it `_remote_umask` or `_default_remote_umask` (see below)?

Reply:
I'll put the remote in braces (remote). It can but does not need to be remote. _default_umask absolutely makes sense.